setup.py

```diff
@@ -49,6 +49,7 @@
     'mpi4py',
     'termcolor',
     'fastcache',
+    'coverage',
```

Comment:
Why do we need to add `coverage` to our dependencies? Pyccel can be installed without coverage, and all tests can also be run without it, correct?

Reply:
Agreed

Reply:
You're right. We should move the installation to the travis script. Should it be in the install section or the before_script section (or the before_install section) ?

Reply:
I would put it in `before_script` as it's only needed by the tests.